app/auth/middleware.test.js

```diff
@@ -73,15 +68,28 @@ describe('Authentication middleware', function() {
       context('when the user locations lookup fails', function() {
         beforeEach(async function() {
           const authentication = proxyquire('./middleware', {
-            '../../common/lib/user': UserStub,
```

Comment:
I think it would be good to keep some kind of stub here to check that the User class is being called with the right properties in the object sent to it.